projects/schematics/src/migrations/4_0/removed-public-api-deprecations/removed-public-api-deprecation.ts

```diff
@@ -250,253 +167,6 @@ export const REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_SEARCH_FAIL,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_SEARCH_FAIL}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_SEARCH_FAIL_CLASS,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_SEARCH_FAIL_CLASS}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_SEARCH_SUCCESS,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_SEARCH_SUCCESS}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_SEARCH_SUCCESS_CLASS,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_SEARCH_SUCCESS_CLASS}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_SEARCH_RESET,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_SEARCH_RESET}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_SEARCH_RESET_CLASS,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_SEARCH_RESET_CLASS}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_ACTION,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_ACTION}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/logout-agent.action.ts
-  {
-    node: LOGOUT_CUSTOMER_SUPPORT_AGENT,
-    importPath: SPARTACUS_CORE,
-    comment: `'${LOGOUT_CUSTOMER_SUPPORT_AGENT}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/logout-agent.action.ts
-  {
-    node: LOGOUT_CUSTOMER_SUPPORT_AGENT_CLASS,
-    importPath: SPARTACUS_CORE,
-    comment: `'${LOGOUT_CUSTOMER_SUPPORT_AGENT_CLASS}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/asm-state.ts
-  {
-    node: ASM_FEATURE,
-    importPath: SPARTACUS_CORE,
-    comment: `'${ASM_FEATURE}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/asm-state.ts
-  {
-    node: CUSTOMER_SEARCH_DATA,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_SEARCH_DATA}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/asm-state.ts
-  {
-    node: STATE_WITH_ASM,
-    importPath: SPARTACUS_CORE,
-    comment: `'${STATE_WITH_ASM}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/asm-state.ts
-  {
-    node: ASM_STATE,
-    importPath: SPARTACUS_CORE,
-    comment: `'${ASM_STATE}' was moved to @spartacus/asm/core.`,
-  },
-
-  // projects/core/src/asm/store/selectors/asm-ui.selectors.ts
-  {
-    node: GET_ASM_UI,
-    importPath: SPARTACUS_CORE,
-    comment: `'${GET_ASM_UI}' was moved to @spartacus/asm/core.`,
-  },
```

Comment:
Same story with selectors. It won't work because they all are under `AsmSelectors`